### PR TITLE
fix(@aws-amplify/aws-amplify-react-native): fix dev build for Windows

### PR DESCRIPTION
**Pull request by david-mcafee**
#### Description of changes
Update to `packages/aws-amplify-react-native/.eslintrc.js ` to fix broken dev build on Windows machines

#### Issue #, if available
#9328 

#### Description of how you validated changes
- Ran successful build on Windows
- Ran successful build on Mac

#### Checklist

- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/aws-amplify-react-native/.eslintrc.js [hunk 1]
```diff
@@ -61,7 +61,12 @@ module.exports = {
 				object: true,
 			},
 		],
-		'prettier/prettier': ['error'],
+		'prettier/prettier': [
+			'error',
+			{
+				'endOfLine': 'auto',
+			}
+		],
```
Comment (calebpollman) [inline packages/aws-amplify-react-native/.eslintrc.js:69]: Nit: condense to one line to follow existing patterns e.g. 
`'react/jsx-wrap-multilines': ['error', { declaration: 'ignore' }],`

**Review by calebpollman — APPROVED** [review]
LGTM, just left a nit

File: packages/aws-amplify-react-native/.eslintrc.js (hunk 1 above)
Comment (david-mcafee) [reply packages/aws-amplify-react-native/.eslintrc.js:69]: Good point, thanks for the review!

**Review by sammartinez — APPROVED** [review]
LGTM 🌮

**Review by Ashish-Nanda — APPROVED** [review]
LGTM

**Pull request merged**